scripts/packaging/createupmpackages.ps1

```diff
@@ -190,14 +151,12 @@ foreach ($entry in $packages.GetEnumerator()) {
     }
 
     Write-Output "======================="
-    Write-Output "Creating $scope.$product.$packageName and publishing to $registryName"
+    Write-Output "Creating $scope.$product.$packageName"
     Write-Output "======================="
-    Start-Process -FilePath $cmdFullPath -ArgumentList "/c $npmPath $npmCommand" -NoNewWindow -Wait
+    Start-Process -FilePath $cmdFullPath -ArgumentList "/c $npmPath pack" -NoNewWindow -Wait

```

Comment:
1. Does this need to run in cmd vs staying in the existing Powershell process?
1. Can it just call `$npmPath pack` directly, instead of spawning a new process?

Reply:
most likely :)

Reply:
i'll give it a try